eng/Version.Details.xml

```diff
@@ -288,17 +288,17 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>c44dc40b763b7c74012622a0a6120cd8ffa35ce4</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.GenAPI" Version="5.0.0-beta.20261.9">
+    <Dependency Name="Microsoft.DotNet.GenAPI" Version="5.0.0-beta.20277.12">
```

Comment:
Ignore everything from the Arcade update commit [fc607e97fc3c](https://github.com/dotnet/aspnetcore/pull/22017/commits/fc607e97fc3c2e9bd1158ea4b7b56bbab8c73515). Arcade should be promoting a build containing [fc3b1e63de02](https://github.com/dotnet/arcade/commit/fc3b1e63de02e2ab97e638959728c9a98dfbac11) real soon now.